inc/assets.php

```diff
@@ -80,25 +81,32 @@ function enqueue_editor_assets() : void {
 		build_file_uri( 'editor.css' ),
 		[
 			'dashicons'
-		]
+		],
+		filemtime( build_file_path( 'editor.css' ) )
 	);
 }
 
 /**
  * Enqueue these assets only on the frontend.
  */
-function enqueue_frontend_assets() : void {
+function enqueue_frontend_scripts() : void {
 	Asset_Loader\enqueue_script_asset(
 		'annual-report-plugin-frontend',
 		build_file_path( 'frontend.asset.php' ),
 		build_file_uri( 'frontend.js' )
 	);
+}
 
+/**
+ * Enqueue these assets in the editor and the frontend.
+ */
+function enqueue_frontend_styles() : void {
 	wp_enqueue_style(
 		'annual-report-plugin-frontend',
 		build_file_uri( 'frontend.css' ),
 		[
 			'dashicons'
-		]
+		],
+		filemtime( build_file_path( 'frontend.css' ) )
```

Comment:
Does `filemtime` work as expected on VIP? I wouldn't expect it to, because there could be multiple application servers, and also because the assets might be going through nginx concatenation.